runatlantis.io/docs/using-atlantis.md

```diff
@@ -124,6 +124,36 @@ Because Atlantis under the hood is running `terraform apply plan.tfplan`, any Te
 They're ignored because they can't be specified for an already generated planfile.
 If you would like to specify these flags, do it while running `atlantis plan`.
 
+---
+## atlantis import
+
+```bash
+atlantis import [options] ADDRESS ID -- [terraform import flags]
```

Comment:
ADDRESS ID is a common name in terraform.

<img width="416" alt="image" src="https://user-images.githubusercontent.com/915731/208832235-5e7f4d41-6fd3-4fa7-bdd4-907c69b42b8f.png">


https://developer.hashicorp.com/terraform/cli/commands/import

Reply:
Nice work in keeping this consistent!